contentcuration/contentcuration/view/internal_views.py

```diff
@@ -37,6 +45,30 @@ def authenticate_user_internal(request):
 @api_view(['POST'])
 @authentication_classes((TokenAuthentication,))
 @permission_classes((IsAuthenticated,))
+def check_version(request):
+    """ Get version of Ricecooker with which CC is compatible """
+    logging.debug("Entering the check_version endpoint")
+    version = json.loads(request.body)['version']
+    status = None
+
+    if str(version) >= VERSION_OK[0]:
+        status = VERSION_OK
+    elif str(version) >= VERSION_SOFT_WARNING[0]:
```

Comment:
Use `LooseVersion` for comparing versions: http://epydoc.sourceforge.net/stdlib/distutils.version.LooseVersion-class.html